### PR TITLE
Feature/pandas converter

### PR DESCRIPTION
**Pull request by sarah-segel**

### CODE REVIEW

File: logs/DataFrame/run_1/metadata.csv
Comment (sarah-segel) [inline logs/DataFrame/run_1/metadata.csv:1]: I'd slightly prefer to get rid of this additional metadata.csv and instead read the objectives from the trials.csv (similar as done for AMLTK when reading from the history.parquet). For run name, I'd use the name of the directory.

File: logs/DataFrame/run_1/configspace.csv
Comment (sarah-segel) [inline logs/DataFrame/run_1/configspace.csv:1]: Would add 1-2 more example runs (maybe also containing some categoricals) to give users some examples.

Comment (sarah-segel) [reply logs/DataFrame/run_1/configspace.csv:1]: We also should add some documentation on the dataframe converter.

File: logs/DataFrame/run_1/metadata.csv
Comment (LukasFehring) [reply logs/DataFrame/run_1/metadata.csv:1]: To do so we rename the column of the `csv` file accordingly as in AMLTK.
Additionally, we need to add whether its maximized or minimized

**Pull request merged**